### ISSUE TITLE
Mean_class_accuracy higher than top1_accuracy during testing

### ISSUE THREAD

**Issue by jannec** [open-mmlab/mmaction2#1143]
Hi,

dataset -> my own Video Dataset with 4 classes,
networks -> 2 TSM and 2 TSN pretrained on Kinetics400 and Diving48 and finetuned on my dataset ,

I have tested finetuned networks on my validation and test splits with test.py script to get top1 and mean class accuracies.
When testing on validation split, all top1 accuracies were higher than mean class, but when it came to test split, it turnd out to be in reverse, mean class accuracy was slightly higher than top1. What is the cause of this strange behaviour?

**Comment by kennymckormick** [comment]
That's not a problem, I guess. `Mean class accuracy` can be either higher or lower than `Top-1 Accuracy`

**Comment by jannec** [comment]
Hm ok,

So, just for better understanding, what is the formlula used for mean_class_acc, or where can i find it? I was convinced that it is just a simple arithmetic mean.

**Comment by kennymckormick** [comment]
https://github.com/open-mmlab/mmaction2/blob/777546f27f8f5a3c83e10d966e2149be2fc9fa31/mmaction/core/evaluation/accuracy.py#L68

**Comment by jannec** [comment]
Thank you, for support :)

**Comment by kennymckormick** [comment]
Please consider giving us a STAR if you found that codebase helpful 😄
